seed/java-sdk/java-inline-path-parameters/java-inline-path-parameters/src/main/java/com/seed/api/resources/imdb/ImdbClient.java

```diff
@@ -0,0 +1,127 @@
+import com.seed.api.resources.imdb.types.Movie;
+import java.io.IOException;
+import okhttp3.Headers;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+public class ImdbClient {
+    protected final ClientOptions clientOptions;
+
+    public ImdbClient(ClientOptions clientOptions) {
+        this.clientOptions = clientOptions;
+    }
+
+    /**
+     * Add a movie to the database
+     */
+    public String createMovie(CreateMovieRequest request) {
+        return createMovie(request, null);
+    }
+
+    /**
+     * Add a movie to the database
+     */
+    public String createMovie(CreateMovieRequest request, RequestOptions requestOptions) {
+        HttpUrl httpUrl = HttpUrl.parse(this.clientOptions.environment().getUrl())
+                .newBuilder()
+                .addPathSegments("movies")
+                .addPathSegments("create-movie")
+                .build();
+        RequestBody body;
+        try {
+            body = RequestBody.create(
+                    ObjectMappers.JSON_MAPPER.writeValueAsBytes(request), MediaTypes.APPLICATION_JSON);
+        } catch (JsonProcessingException e) {
+            throw new SeedApiException("Failed to serialize request", e);
+        }
+        Request okhttpRequest = new Request.Builder()
+                .url(httpUrl)
+                .method("POST", body)
+                .headers(Headers.of(clientOptions.headers(requestOptions)))
+                .addHeader("Content-Type", "application/json")
+                .build();
+        OkHttpClient client = clientOptions.httpClient();
+        if (requestOptions != null && requestOptions.getTimeout().isPresent()) {
+            client = clientOptions.httpClientWithTimeout(requestOptions);
+        }
+        try (Response response = client.newCall(okhttpRequest).execute()) {
+            ResponseBody responseBody = response.body();
+            if (response.isSuccessful()) {
+                return ObjectMappers.JSON_MAPPER.readValue(responseBody.string(), String.class);
+            }
+            String responseBodyString = responseBody != null ? responseBody.string() : "{}";
+            throw new SeedApiApiException(
+                    "Error with status code " + response.code(),
+                    response.code(),
+                    ObjectMappers.JSON_MAPPER.readValue(responseBodyString, Object.class));
+        } catch (IOException e) {
+            throw new SeedApiException("Network error executing HTTP request", e);
+        }
+    }
+
+    public Movie getMovie(String movieId) {
+        return getMovie(movieId, GetMovieRequest.builder().build());
+    }
+
+    public Movie getMovie(String movieId, GetMovieRequest request) {
+        return getMovie(movieId, request, null);
+    }
+
+    public Movie getMovie(String movieId, GetMovieRequest request, RequestOptions requestOptions) {
```

Comment:
@amckinney Looks like this was already implemented. If we want, we can make this test more general and run it on other generators to see which ones we're missing to support this feature.

Reply:
In this case, we actually want `movieId` to be included in `GetMovieRequest`. That's what in-lined in the request is referring to.

Reply:
Gotcha, will get on this then!